poc_iot_verifier/src/main.rs

```diff
@@ -1,38 +1,85 @@
-use poc_iot_verifier::{loader, mk_db_pool, purger, runner, Result};
+use clap::Parser;
+use poc_iot_verifier::{loader, purger, runner, Result, Settings};
+use std::path;
 use tokio::signal;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
+#[derive(Debug, clap::Parser)]
+#[clap(version = env!("CARGO_PKG_VERSION"))]
+#[clap(about = "Helium POC IOT Verifier")]
+pub struct Cli {
+    /// Optional configuration file to use. If present the toml file at the
+    /// given path will be loaded. Environemnt variables can override the
+    /// settins in the given file.
+    #[clap(short = 'c')]
+    config: Option<path::PathBuf>,
+
+    #[clap(subcommand)]
+    cmd: Cmd,
+}
+
+impl Cli {
+    pub async fn run(self) -> Result {
+        let settings = Settings::new(self.config)?;
+        tracing_subscriber::registry()
+            .with(tracing_subscriber::EnvFilter::new(&settings.log))
+            .with(tracing_subscriber::fmt::layer())
+            .init();
```

Comment:
is this redundant with the one defined in the `server` subcommand? if you want the tracing subscriber here to allow logging to std when running one-off commands can we remove the one in the server subcommand?

Reply:
@madninja You got an reasoning for this, or just an oversight?